modules/javafx.controls/src/main/java/javafx/scene/control/MultipleSelectionModelBase.java

```diff
@@ -876,8 +885,40 @@ public int bitsetSize() {
         }
 
         @Override public int indexOf(Object obj) {
-            reset();
-            return super.indexOf(obj);
```

Comment:
`reset()` was previously called since `super.indexof()` was making calls to `get(int index)` and it needed to have `lastGetIndex` and `lastGetValue` set to -1